src/index.tsx

```diff
@@ -5,19 +5,117 @@ import App from "./components/App";
 import ErrorBoundary from "./components/ErrorBoundary";
 import { Provider } from "react-redux";
 import { store } from "./redux/store";
+import { createHashRouter, redirect, RouterProvider } from "react-router-dom";
+import NotFound from "./components/common/Error/NotFound";
 
 import "@fontsource/montserrat";
 
 import "./index.css";
 
+const router = createHashRouter([
```

Comment:
Why are you using `createHashRouter` it is not recommended, and that is why you have this in url
![image](https://user-images.githubusercontent.com/106384184/211561111-f40739a5-7a86-42df-b63e-0d824c38a40e.png)

please use [createBrowserRouter](https://reactrouter.com/en/main/routers/create-browser-router) instead